build-hyperv.ps1

```diff
@@ -1,20 +1,45 @@
 #!/usr/local/bin/pwsh
 param(
   [switch]$offline,
-  [switch]$SkipPester
+  [switch]$SkipPester,
+  [switch]$ServerOnly,
+  [switch]$TentacleOnly
 )
 
 . Tests/powershell-helpers.ps1
 
 Start-Transcript .\vagrant-hyperv.log -Append
 
+# Clear the OctopusDSCTestMode Env Var
```

Comment:
Consider moving all this new code to a function.

Also consider moving the opening brace onto the same line as the `if`, to make it more concise (controversial i know)